stdlib/2and3/asynchat.pyi

```diff
@@ -18,12 +18,12 @@ class async_chat(asyncore.dispatcher):
     def collect_incoming_data(self, data: bytes) -> None: ...
     @abstractmethod
     def found_terminator(self) -> None: ...
-    def set_terminator(self, term: Union[str, int, None]) -> None: ...
-    def get_terminator(self) -> Union[str, int, None]: ...
+    def set_terminator(self, term: Union[bytes, int, None]) -> None: ...
```

Comment:
That flag would have to be set by the client of this code. I'm pretty sure that no one ever has set it, nor ever will – it's broken in at least three different ways, possibly four.